### DISCUSSION TITLE
Follow developer

### DISCUSSION THREAD

**Discussion by pontifezo** [chr233/ASFEnhance#160] [功能建议 / Ideas]
Please add commands to follow/unfollow the developer page.

For example:
https://store.steampowered.com/developer/klei
https://store.steampowered.com/developer/Tripwire

**Comment by chr233** [comment]
you can use `followcurator 112393` to follow developer Klei.
112393 can be found in traffic.

![image](https://github.com/chr233/ASFEnhance/assets/13798576/7a7303a2-3519-41ba-9ace-7bb3be931642)
